### PR TITLE
UPDATED README

### PR DESCRIPTION
**Pull request by atul-saswat**
I have made the readme beginner friendly and the steps have been also illustrated for the newbie to understand alongside the code.

### CODE REVIEW

**Comment by atul-saswat** [comment]
please check my pull request

File: README.md [hunk 1]
```diff
@@ -1,12 +1,43 @@
-# fetch-topicals
+# FETCH TOPICALS (AMUL)
```
Comment (vineetjc) [inline README.md]: No need of this line, it's not adding anything if we're illustrating below

File: README.md [hunk 2]
```diff
@@ -1,12 +1,43 @@
...
+
 [![Join the chat at https://gitter.im/fetch-amul-topicals/Lobby](https://badges.gitter.im/fetch-amul-topicals/Lobby.svg)](https://gitter.im/fetch-amul-topicals/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-A Python script that fetches details all ‘topicals’ across all years from the Amul website (http://www.amul.com/m/amul-hits) and has the feature to download the pictures by year into your local system
+It uses a Python script (aka **web scraper**) that fetches details (here ‘topicals’) across all years from the Amul's website (http://www.amul.com/m/amul-hits) and has the feature to download the pictures chronologically into your local system.
```
Comment (vineetjc) [inline README.md]: It is a Python script in itself. This is an unnecessary expansion of the already existing description. Chuck preferably.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,43 @@
...
+
+### Fundamentals of Web Scraping
```
Comment (vineetjc) [inline README.md]: We don't need to explain this, this is very generic information.

File: README.md [hunk 4]
```diff
@@ -1,12 +1,43 @@
...
+Web scraping is a computer software technique of extracting information from websites. This technique mostly focuses on the transformation of unstructured data (HTML format) on the web into structured data (database or spreadsheet). <br>
+Tips:
+Get familiar with the HTML tags ; CSS, Layout & Architecture to extract the data in a better manner
+
+### Steps
```
Comment (vineetjc) [inline README.md]: Either we should expand on the steps of installation and usage, or don't use this at all. This 'source code reading' part is understood by reading the code.

File: README.md [hunk 5]
```diff
@@ -1,12 +1,43 @@
...
+2.Go to the website <br>
+3.Find the sequence in the URL of the images. Usually they are in consecutive numbers <br>
+4.Use the method of webscraping <br>
+5.Use try-catch blocks for exception might arise in between <br>
 
 ## Amul Website
 ![AMUL website screenshot](images/amul.png)
+<br>
+**Finding the Sequence :**<br>
+If you check for each year the image URL reads "http://www.amul.com/m/amul-hits?s=####" where #### represents the year number which goes way back to 1976 from 2018 (as of Dec 2018)<br>
```
Comment (vineetjc) [inline README.md]: This we can't keep because things can change with time and systems can change.

File: README.md [hunk 6]
```diff
@@ -1,12 +1,43 @@
...
+Try pasting this in Address Bar: http://www.amul.com/m/amul-hits?s=2018  to get redirected to the 2018 Amul Hits<br>
+
+## Zooming into each year
```
Comment (vineetjc) [inline README.md]: As said above, can't keep this too because the system can change and we'd need to update our scraper itself.

File: README.md [hunk 7]
```diff
@@ -1,12 +1,43 @@
...
+![AMUL website screenshot](images/amul_1.png) <br>
+**Finding the Sequence :**<br>
+If you check for the individual years the image URL reads "http://www.amul.com/files/hits/amul-hits-####.jpg"<br>
+Substitute #### for: <br>
+2018 : 2651 to 2660 <br>
```
Comment (vineetjc) [inline README.md]: These numbers aren't permanent. It changes with the addition of new images.

File: README.md [hunk 8]
```diff
@@ -1,12 +1,43 @@
...
+2017 : 2504 to 2513 <br>
+. <br>
+. <br>
+1976 : 1245 to 1251 (This can be seen in the executed code) <br>
 
 ## Executing the Code
 ![Terminal Output](images/terminal.png)
+
+With the sequence found out, we can execute the required program inside a loop and scrape our favorite AMULLICIOUS topicals and make a comic strip out of it (maybe!).
```
Comment (vineetjc) [inline README.md]: ~AMULICIOUS~
Please omit this too.

File: README.md [hunk 9]
```diff
@@ -1,12 +1,48 @@
...
+
 [![Join the chat at https://gitter.im/fetch-amul-topicals/Lobby](https://badges.gitter.im/fetch-amul-topicals/Lobby.svg)](https://gitter.im/fetch-amul-topicals/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-A Python script that fetches details all ‘topicals’ across all years from the Amul website (http://www.amul.com/m/amul-hits) and has the feature to download the pictures by year into your local system
```
Comment (vineetjc) [inline README.md]: Use this text only "A Python script that fetches...." instead of your "It uses a Python script...."

File: README.md [hunk 10]
```diff
@@ -1,12 +1,48 @@
...
 ![AMUL website screenshot](images/amul.png)
+<br>
 
 ## Executing the Code
 ![Terminal Output](images/terminal.png)
+
+##License  
+The code in this project is licensed under the MIT License 2016 - Kartik Singhal.
+
+With the sequence found out, we can execute the required program inside a loop and scrape our favorite topicals and make a comic strip out of it (maybe!).
```
Comment (vineetjc) [inline README.md]: Chuck this last line. No need.

File: README.md [hunk 11]
```diff
@@ -1,12 +1,48 @@
...
 
-## Amul Website
+### Requirements
+* Programming Language : Ruby/ Python (2 or 3)
+* Library : Beautiful Soup/ Scrapy/ Selenium/Python Requests/ LXML/ Urlib
+
+##Installation Guide
+###Setup instructions:
+* Fork the repository, and clone your fork into your machine.
+ * Use pip-install to install all required modules, for example pipi install beautifulsoup4
```
Comment (vineetjc) [inline README.md]: Put code snippets/commands like this: `pip install bs4` (for example)

File: README.md [hunk 12]
```diff
@@ -1,12 +1,48 @@
...
+
+### Steps
+1.Import necessary libraries <br>
+2.Go to the website <br>
+3.Find the sequence in the URL of the images. Usually they are in consecutive numbers <br>
+4.Use the method of webscraping <br>
+5.Use try-catch blocks for exception might arise in between <br>
 
-## Amul Website
+### Requirements
```
Comment (vineetjc) [inline README.md]: Requirements aren't correct, we don't use Ruby for example. Please correct this and refer to the code for the exact libraries used

File: README.md (hunk 4 above)
Comment (vineetjc) [reply README.md]: Chuck your steps 1-5. No need.

File: README.md [hunk 13]
```diff
@@ -1,12 +1,50 @@
 # fetch-topicals
 ![forthebadge](https://forthebadge.com/images/badges/made-with-python.svg)
+
```
Comment (vineetjc) [inline README.md]: What are these blank lines? No need.

File: README.md (hunk 12 above)
Comment (vineetjc) [reply README.md]: - Chuck those libraries that are already there when one installs Python. (like os, sys).
- What is 'import'?

**Pull request closed without merging**